Game.h

```diff
@@ -11,6 +11,9 @@
 #include <Looper.h>
 #include <SupportDefs.h>
 
+#define HIGHSCORE_FILENAME "Highscore.dat"
```

Comment:
We're not used to file suffixes in Haiku (at least for ususally not shared files). Just call it "Highscore".